src/core_ocean/shared/mpas_ocn_vmix.F

```diff
@@ -319,7 +319,7 @@ subroutine ocn_vel_vmix_tend_implicit_rayleigh(meshPool, dt, kineticEnergyCell,
          ! Apply bottom drag boundary condition on the viscous term
          ! second line uses sqrt(2.0*kineticEnergyEdge(k,iEdge))
          B(N) = 1.0_RKIND - A(N) + dt*implicitBottomDragCoef &
-              * sqrt(kineticEnergyCell(k,cell1) + kineticEnergyCell(k,cell2)) / layerThicknessEdge(k,iEdge) &
+              * sqrt(kineticEnergyCell(N,cell1) + kineticEnergyCell(N,cell2)) / layerThicknessEdge(N,iEdge) &
```

Comment:
@vanroekel and @jonbob, this is an important correction, where the index for kinetic energy in bottom drag was effectively using `N-1` rather than the bottom layer `N`.  It will be a non-bfb change in E3SM. It should be a small change, but will need to be tested.

Reply:
@mark-petersen and @jonbob once anvil is back today I can easily test this in a G-case